src/components/PostList/PostList.jsx

```diff
@@ -1 +1,15 @@
-export const PostList = () => <>Put the list here</>;
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable prettier/prettier */
```

Comment:
Consider removing unnecessary ESLint disable comments if they are not needed. This will help keep the code clean and maintainable.